diff --git a/lib/dashing/app.rb b/lib/dashing/app.rb
index a2d0b2f..015ab4a 100644
--- a/lib/dashing/app.rb
+++ b/lib/dashing/app.rb
@@ -44,7 +44,7 @@ def authenticated?(token)
 set :template_languages, %i[html erb]
 
 if File.exist?(settings.history_file)
-  set :history, YAML.load_file(settings.history_file)
+  set :history, YAML.load_file(settings.history_file, fallback: {})
 else
   set :history, {}
 end
